docs/source/api/tutorials/In Depth 3 - Drawing DiHypergraphs.ipynb

```diff
@@ -13,7 +13,7 @@
    "id": "ea8aad6a",
```

Comment:
<p>same here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/xgi-org/xgi/pull/492/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Reply:
This example is now removed.